src/control/server/instance_storage.go

```diff
@@ -99,6 +99,23 @@ func (ei *EngineInstance) awaitStorageReady(ctx context.Context, skipMissingSupe
 	}
 	ei.log.Debugf("needsScmFormat: %t", needsScmFormat)
 
+	if !needsMetaFormat && ei.storage.ControlMetadataHasPath() && needsScmFormat {
```

Comment:
I'd like to understand what this use case actually means. What is the situation where a user unmounts their tmpfs but doesn't want to reformat the whole system?

Reply:
When control Metadata is intact (on dedicated storage) after a reboot which has cleared tmpfs.

Reply:
There is a slight change in the logic now:
1. If metadata is hosted on the same tmpfs mount then , daos_server restart will reuse the mount and starts the engines.
2. If metadata is hosted in a different path (recommended for md_on_ssd), then tmpfs is force remounted every time the daos_server is restarted. The force remount will cleanup leftover files and directories in the previous run.
3. If the metadata is hosted on a different path and tmpfs is unmounted (node restart), then daos_server will mount the tmpfs and continue with the engine start. This case is covered by (2).

Reply:
Thanks for the explanations, @sherintg and @tanabarr. If the control metadata is on the same tmpfs, and the tmpfs isn't mounted, will that still trigger a need to format? In my view there is value in keeping that path alive (specifically in allowing us to test that flow without SCM).

Reply:
Yes. If tmpfs is not mounted then you lose all of the control metadata if it was earlier hosted on tmpfs.